src/dialogs/components/dialog/dialogField.ts

```diff
@@ -0,0 +1,81 @@
+import {DialogFieldClass} from '../../interfaces/abstractDialogFieldClass';
+import * as _ from 'lodash';
+/**
+ * This component deals with an individual dialog field
+ *
+ * @extends miqStaticAssets.dialog.DialogFieldClass
+ * @memberof miqStaticAssets.dialog
+ * @ngdoc controller
+ * @name DialogFieldController
+ */
+
+export class DialogFieldController extends DialogFieldClass {
+     public service: any;
```

Comment:
This made it past whatever linting rules are in place in the repo.  I can switch this to two spaces though.